safegraph/delphi_safegraph/run.py

```diff
@@ -24,11 +24,14 @@ def run_module():
     aws_endpoint = params["aws_endpoint"]
     wip_signal = params["wip_signal"]
 
-    process_file = partial(process,
-                           signal_names=add_prefix(SIGNALS, wip_signal, prefix='wip_'),
-                           geo_resolutions=GEO_RESOLUTIONS,
-                           export_dir=export_dir,
-                           )
+    def process_file(current_filename):
```

Comment:
It looks like this is changing the default behavior -- I don't know how Delphi handles changes like this but we would typically put it behind a flag -- is something like that worth doing, or are Delphi ok with just manually running and verifying?

Reply:
Unsure what the protocol is

Reply:
@krivard can you help advise?

Reply:
Protocol is: If it's going to change the behavior of the _output_, then it needs to be more broadly reviewed for statistical quality, and either mentioned in the public changelog (if it's a minor change) or under a new signal name (if it's a major change). If it's a change in how the code works, then if it gets called by another system, put it behind a flag. If it's just called internally, document it, but otherwise anything goes.

It looks like this particular change falls mostly under the latter. The new 7dav signals will get statistically reviewed before we release them, but just changing `process_file` to include a batch in addition to a single file is fine. Since we are promoting it to a full function instead of a partial/lambda, would you add a docstring that notes the constraints imposed by `process()`?

Reply:
Done.